resources/plugins/test/style.css

```diff
@@ -1,7 +1,3 @@
-.bjs-powered-by {
```

Comment:
Why this change?

Reply:
The bpmn.io watermark was removed in this [commit](https://github.com/camunda/camunda-modeler/commit/b1102b4e43db10aa56923816b929f826345c1c67). However, this `block` forces the watermark to remain visible in the modeler and allows it to gain focus when navigating via keyboard tabbing.
<img width="544" alt="Screenshot 2024-12-09 at 14 52 51" src="https://github.com/user-attachments/assets/ebbe154c-8520-46ff-9c32-8924a526ad2f">